client/src/panels/internal_services/isc.yaml

```diff
@@ -7,7 +7,7 @@ internal_service_panel_text:
   en: |
     Last year ({{pa_last_year}}), **{{fmt_percentage1 isc_fte_pct}}** of {{gl_tt "employees" "FTE"}} in {{subj_name subject}} worked for internal service programs. In comparison, government as a whole uses **{{fmt_percentage1 gov_isc_fte_pct}}** of its workforce for internal services.
   fr: |
-    L'an dernier ({{pa_last_year}}), **{{fmt_percentage1 isc_fte_pct}}** des {{gl_tt "FTE" "employés"}} dans l'organisation {{subj_name subject}} travaillaient pour des programmes de services internes.
+    L'an dernier ({{pa_last_year}}), **{{fmt_percentage1 isc_fte_pct}}** des {{gl_tt "employés" "FTE"}} dans l'organisation {{subj_name subject}} travaillaient pour des programmes de services internes.
```

Comment:
Need to add the second sentence in French:

"En comparaison, **{{fmt_percentage1 gov_isc_fte_pct}}** de l’effectif de le gouvernement travaillaient pour des programmes de services internes."

Reply:
Another thing to note here from Val: Wording of `Last year (2017-18)` should change to `In 2017-18`. Just note for myself to fix